packages/ingest/src/granule.ts

```diff
@@ -137,14 +185,23 @@ async function moveGranuleFileWithVersioning(source, target, sourceChecksumObjec
  * @throws {DuplicateFile} DuplicateFile error in `error` case.
  * @returns {Array<Object>} List of file version S3 Objects in `version` case, otherwise empty.
  */
-async function handleDuplicateFile({
-  source,
-  target,
-  duplicateHandling,
-  checksumFunction,
-  syncFileFunction,
-  ACL
-}) {
+export async function handleDuplicateFile(params: {
+  source: { Bucket: string, Key: string },
+  target: { Bucket: string, Key: string },
+  duplicateHandling: DuplicateHandling,
+  checksumFunction?: (bucket: string, key: string) => Promise<[string, string]>,
+  syncFileFunction?: (bucket: string, key: string) => Promise<void>,
+  ACL?: string
+}): Promise<{ Bucket: string, Key: string, size: number }[]> {
```

Comment:
Should use the same interface that we extract from the above function

Reply:
https://github.com/nasa/cumulus/pull/1798/commits/08ceba70690ea61f11cecfef5c6a71d1e1c13c47#diff-8b192e80bd09d39c934ec14baef99981R47